src/frontend/web_application/src/components/MessageList/components/MessageActionsContainer/index.jsx

```diff
@@ -1,38 +1,91 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import Dropdown, { withDropdownControl } from '../../../../components/Dropdown';
+import { CheckboxFieldGroup } from '../../../form';
 import Button from '../../../Button';
+import Icon from '../../../Icon';
+
 import './style.scss';
 
-const MessageActionsContainer = ({ message, onDelete, __, className, ...props }) => {
-  const messageActionsContainerClassName = classnames(
-    'm-message-actions-container',
-    className,
-  );
+const DropdownControl = withDropdownControl(Button);
+
+function generateStateFromProps(props) {
+  const { message } = props;
+
+  return {
+    isRead: !message.is_unread,
```

Comment:
I understand and I'm agree here to copy `isRead` in state because it is required by the checkbox to appears checked when user click on it.